docs/enterprise-setup/self-managed/implementation-guide.md

```diff
@@ -0,0 +1,103 @@
+                <li>Use <code>kubectl config get-contexts</code> to show the contexts available.</li>
+                <li>Run <code>kubectl config use-context $EKS_CONTEXT</code> to access the cluster with kubectl.</li>
+            </ol>
+        </TabItem>
+    </Tabs>
+</details>
+
+## Deploy Airbyte Self Managed
+
+### Add Airbyte Helm Repository
+
+Follow these instructions to add the Airbyte helm repository:
+1. Run `helm repo add airbyte https://airbytehq.github.io/helm-charts`, where `airbyte` is the name of the repository that will be indexed locally.
+2. Perform the repo indexing process, and ensure your helm repository is up-to-date by running `helm repo update`.
+3. You can then browse all charts uploaded to your repository by running `helm search repo airbyte`.
+
+### Clone & Configure Airbyte
+
+
+1. `git clone` the latest revision of the [airbyte-platform repository](https://github.com/airbytehq/airbyte-platform)
+
+2. Create a new `airbyte.yml` file in the `configs` directory of the `airbyte-platform` folder. You may also copy `airbyte.sample.yml` to use as a template:
+
+```sh
+cp configs/airbyte.sample.yml configs/airbyte.yml
+```
+
+3. Add your Airbyte Enterprise license key to your `airbyte.yml`. 
+
+4. Add your [auth details](/airbyte-enterprise#single-sign-on-sso) to your `airbyte.yml`. Auth configurations aren't easy to modify after Airbyte is installed, so please double check them to make sure they're accurate before proceeding.
+
+<details>
+    <summary>Configuring auth in your airbyte.yml file</summary>
+
+To configure SSO with Okta, add the following at the end of your `airbyte.yml` file:
+
+```
+auth:   
+    identity-providers:
+        -   type: okta
+            domain: $OKTA_DOMAIN
+            app-name: $OKTA_APP_INTEGRATION_NAME
+            client-id: $OKTA_CLIENT_ID
+            client-secret: $OKTA_CLIENT_SECRET
+```
+
+To configure basic auth (deploy without SSO), remove the entire `auth:` section from your airbyte.yml config file. You will authenticate with the instance admin user and password included in the your `airbyte.yml`.
+
+</details>
+
+### Install Airbyte Self Managed
+
+Install Airbyte Enterprise on helm using the following command:
+
+```text
+./tools/bin/install_airbyte_pro_on_helm.sh
+```
+
+The default release name is `airbyte-pro`. You can change this via the `RELEASE_NAME` environment
+variable.
+
+### Customizing your Airbyte Self Managed Deployment
+
+In order to customize your deployment, you need to create `values.yaml` file in a local folder and populate it with default configuration override values. A `values.yaml` example can be located in [charts/airbyte](https://github.com/airbytehq/airbyte-platform/blob/main/charts/airbyte/values.yaml) folder of the Airbyte repository.
+
+After specifying your own configuration, run the following command:
+
+```text
+./tools/bin/install_airbyte_pro_on_helm.sh --values path/to/values.yaml %release_name% airbyte/airbyte
```

Comment:
I know this was in the previous docs as well, but using `%` here is incorrect.  We should have `$RELEASE_NAME` or simply leave it out.